pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -1100,7 +1100,7 @@ func TestChangefeedTimestamps(t *testing.T) {
 		sqlDB.Exec(t, `CREATE TABLE foo (a INT PRIMARY KEY)`)
 		sqlDB.Exec(t, `INSERT INTO foo VALUES (0)`)
 
-		foo := feed(t, f, `CREATE CHANGEFEED FOR foo WITH updated, resolved`)
+		foo := feed(t, f, `CREATE CHANGEFEED FOR foo WITH updated, resolved`, optOutOfMetamorphicEnrichedEnvelope{reason: "this test calls testfeed.Next() directly"})
```

Comment:
could you elaborate on why calling Next directly is a problem?

Reply:
most of the meat of this feature is inside assertMessages, which is more commonly used. calling Next() directly bypasses all of that, and for simplicity i'm not supporting it